WebDevStudios/WDS-React-Post-Search
# Feature/add ie11 support

Adds support for IE11 by including `isomorphic-fetch` in `package.json` and in `searchForm.js`.

### To Test ###
1. Test searching in all browsers to ensure nothing has stopped working. In my testing, I have run into no issues so far.
2. Test searching in IE11

This is working on my local as seen here:
https://app.crossbrowsertesting.com/public/i29947718af4369d/livetests/15987664/videos/z10273d94da41f953fbb
